### PR TITLE
Replace link into main branch of Identity and Consent Management within "Authorization and authentication" text template

### LINKED ISSUE #185: Link within mandatory text for all APIs points to main branch - need to be corrected in rc.2
**Problem description**

The [mandatory template text](https://github.com/camaraproject/IdentityAndConsentManagement/blob/v0.2.0-rc.1/documentation/CAMARA-API-access-and-user-consent.md#mandatory-template-for-infodescription-in-camara-api-specs) which got copied into every API Version for the meta-release contains a link to the main branch of Identify and Consent Management:

> ### Authorization and authentication
> 
> [Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.

Shortly after the Fall24 meta release this branch will not any more contain the version which is the relevant one for the meta-release. Actually this link would hinder the working group to work on the Profile in the expected way.

This issue was originally brought up by @sfnuser in https://github.com/camaraproject/NumberVerification/pull/121#discussion_r1690204839 and was discussed within https://github.com/camaraproject/ReleaseManagement/issues/51. 

**Expected behavior**

As the preferred short-term solution it was identified that there should be only a link to the ICM repository, not the concrete document. The README of the repository then should guide to the correct released version of the profile.

Text proposal:
```
### Authorization and authentication

The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
```

The README.md of ICM should have a link to the document which is the valid one for the Fall24 meta-release (similar to the links on Commonalities to the most important documents)

**Alternative solution**

Other solutions were discussed in https://github.com/camaraproject/ReleaseManagement/issues/51:
* links to a concrete release tag of ICM wouldn't work as ICM would lose the option to release patch releases of the Profile
* introducing additional "meta" tags was postponed to a later release cycle

**Additional context**

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:

See #185, updating the first paragraph of the mandatory text for all APIs

#### Which issue(s) this PR fixes:

Will only partly fix #185, readme update will be within #181

#### Special notes for reviewers:

I consider this correction as essential for the rc.2

#### Changelog input

```
Replaced link into main branch of Identity and Consent Management within "Authorization and authentication" text template
```

### CODE REVIEW

**Comment by jpengar** [comment]
@hdamker To fix #185, we would need another PR to update the README file as requested. We may do this as part of #181.

**Comment by hdamker** [comment]
> @hdamker To fix #185, we would need another PR to update the README file as requested. We may do this as part of #181.

Yes, I'm working already on suggestion on #181. Have updated the description above.

**Review by jpengar — APPROVED** [review]
LGTM

File: documentation/CAMARA-API-access-and-user-consent.md [hunk 1]
```diff
@@ -355,7 +355,7 @@ The documentation template below must be used as part of the API documentation i
 ```
 ### Authorization and authentication
 
-[Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
+The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
```
Comment (jpengar) [inline documentation/CAMARA-API-access-and-user-consent.md:358]: I guess that the link is not formatted as `[](url)` on purpose, right?

Comment (hdamker) [reply documentation/CAMARA-API-access-and-user-consent.md:358]: Actually I hadn't thought about it, but yes, it makes sense to have the URL openly visible.

Comment (tanjadegroot) [reply documentation/CAMARA-API-access-and-user-consent.md:358]: looks good (OK from Release Mgmt if Herbert did not already have that hat)

**Comment by jpengar** [comment]
@AxelNennker  @sebdewet @tanjadegroot If it's okay with you, I can merge this PR.

**Review by AxelNennker — APPROVED** [review]

**Pull request merged**